packages/mobile/src/app/ErrorMessages.ts

```diff
@@ -27,6 +27,7 @@ export enum ErrorMessages {
   ODIS_QUOTA_ERROR = 'odisQuotaError',
   SALT_FETCH_FAILURE = 'saltFetchFailure',
   SALT_QUOTA_EXCEEDED = 'saltQuotaExceededError',
+  ODIS_INSUFFICIENT_BALANCE = 'odisInsufficientBalance',
```

Comment:
These should correspond to strings in the localized JSON files otherwise if the error message is intercepted and changed (like is done for verification but not other places) the error would just show 'odisInsufficientBalance'